### ISSUE TITLE
NFC 0.69.1 Can not parse file

### ISSUE THREAD

**Issue by Antiarchitect** [flipperdevices/flipperzero-firmware#1904]
### Describe the bug.

I have two phones with NFC chips. Reading one (One Plus 7) goes normally and I can Emulate UID normally. But have issues with another one which is Honor 50: I can read NFC and save it but selecting saved file tells `Can not parse file` and thus I have no actions menu (UX bug) so I cannot even delete it.
![image](https://user-images.githubusercontent.com/50209/197215623-e6c4c238-d313-4af7-ae7c-6685fef3585f.png)
![image](https://user-images.githubusercontent.com/50209/197215656-802e8ea6-c70f-49bd-8576-5e90bbcf2c8b.png)

### Reproduction

1. Go to NFC
2. Select Read
3. Read Honor 50 with NFC turned on
4. Save and name
5. Try to select the file in Saved section

**Comment by gornekich** [comment]
Hello @Antiarchitect !
Could you please share the content of the saved file? Nfc type and UID fields in particular

**Comment by gornekich** [comment]
Not UID, but it's length

**Comment by Antiarchitect** [comment]
Will try to extract the file a bit later :)

**Comment by Antiarchitect** [comment]
[https://dev.flpr.app/s#path=nfc/Vika.nfc&Filetype=Flipper+NFC+device&Version=2&Device+type=UID&UID=00+00+00+00+00+00+00+00+00+00&ATQA=00+00&SAK=00](url)

**Comment by Antiarchitect** [comment]
https://dev.flpr.app/s#path=nfc/Vika.nfc&Filetype=Flipper+NFC+device&Version=2&Device+type=UID&UID=00+00+00+00+00+00+00+00+00+00&ATQA=00+00&SAK=00

**Comment by Antiarchitect** [comment]
```
Filetype: Flipper NFC device
Version: 2
Device type: UID
UID: 00 00 00 00 00 00 00 00 00 00
ATQA: 00 00
SAK: 00
```

**Comment by LowSkillDeveloper** [comment]
I have the same problem with my bus pass.

```
Filetype: Flipper NFC device
Version: 2
# Nfc device type can be UID, Mifare Ultralight, Mifare Classic, Bank card
Device type: Mifare DESFire
# UID, ATQA and SAK are common for all formats
UID: 04 73 43 85 97 7D 80
ATQA: 44 03
SAK: 20
# Mifare DESFire specific data
PICC Version: 04 01 01 01 00 1A 05 04 01 01 01 04 1A 05 04 73 43 85 97 7D 80 B9 0C 22 41 50 05 22
PICC Free Memory: 1312
PICC Change Key ID: 00
PICC Config Changeable: true
PICC Free Create Delete: false
PICC Free Directory List: true
PICC Key Changeable: true
PICC Max Keys: 01
PICC Key 0 Version: 00
Application Count: 10
Application IDs: D0 02 F4 70 02 F0 60 34 F5 10 20 F1 50 89 F8 20 20 F1 80 10 F1 A0 11 F1 0B 10 00 FF FF FF
```

![Screenshot_20221028-182105_Flipper](https://user-images.githubusercontent.com/25121341/198849886-c31c7d7b-35c8-437c-8cd9-de01a9a95861.jpg)

**Comment by mh-** [comment]
> I have the same problem with my bus pass.

Ok, yes that's the same problem as in #2040 - the DESFire import 
https://github.com/flipperdevices/flipperzero-firmware/blob/820afd2aec377629c3e514ceb57ecdd0f9163b6c/lib/nfc/nfc_device.c#L426
expects details about the key for each application.

If you modify the file like this
```
Filetype: Flipper NFC device
Version: 2
# Nfc device type can be UID, Mifare Ultralight, Mifare Classic, Bank card
Device type: Mifare DESFire
# UID, ATQA and SAK are common for all formats
UID: 04 73 43 85 97 7D 80
ATQA: 44 03
SAK: 20
# Mifare DESFire specific data
PICC Version: 04 01 01 01 00 1A 05 04 01 01 01 04 1A 05 04 73 43 85 97 7D 80 B9 0C 22 41 50 05 22
PICC Free Memory: 1312
PICC Change Key ID: 00
PICC Config Changeable: true
PICC Free Create Delete: false
PICC Free Directory List: true
PICC Key Changeable: true
PICC Max Keys: 01
PICC Key 0 Version: 00
Application Count: 0
```
i.e. set Application Count to 0, Flipper can read the file and emulate the UID.

But: I would advise against trying that UID in an actual bus, assuming that you are not the owner of the bus company or authorized to do penetration testing by them. And also, it is unlikely that such a system will accept the clone just based on the UID.

**Comment by skotopes** [comment]
Fixed in #2041 thanks to @mh-
